src/parachains.json

```diff
@@ -0,0 +1,14 @@
+[
+"wss://kusama-rpc.polkadot.io",
```

Comment:
https://syllabus.codeyourfuture.io/guides/code-style-guide

Reply:
applied prettier with the recommended .prettierrc